ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```
@@ -95,3 +95,90 @@ keys:
                         type: int
                         convert_types:
                           - str
+                  police:
+                    type: dict
+                    keys:
+                      cir:
+                        type: dict
+                        description: Set committed information rate.
+                        keys:
+                          cir_rate:
+                            required: true
+                            type: int
+                            convert_types:
+                              - str
+                            description: |
```

This removes the trailing newline.
```suggestion
                            description: |-
```